Source/Core/DolphinWX/Frame.cpp

```diff
@@ -690,6 +690,64 @@ WXLRESULT CFrame::MSWWindowProc(WXUINT nMsg, WXWPARAM wParam, WXLPARAM lParam)
 }
 #endif
 
+void CFrame::InhibitScreensaver()
+{
+// Inhibit the screensaver. Depending on the operating system this may also
+// disable low-power states and/or screen dimming.
+
+#if defined(HAVE_X11) && HAVE_X11
+  if (SConfig::GetInstance().bDisableScreenSaver)
```

Comment:
Yeah, let's just leave it as is for now - but put in a comment that it's also inhibiting suspend on Windows.